zerver/tests/test_audit_log.py

```diff
@@ -25,3 +26,14 @@ def test_user_activation(self):
             .order_by('event_time').values_list('event_type', flat=True))
         self.assertEqual(event_types, ['user_created', 'user_deactivated', 'user_activated',
                                        'user_deactivated', 'user_reactivated'])
+
+class TestChangePassword(ZulipTestCase):
+    def test_change_password(self):
+        # type: () -> None
+        realm = get_realm('zulip')
+        now = timezone.now()
+        user = do_create_user('email', 'password', realm, 'full_name', 'short_name')
+        password = 'test1'
+        do_change_password(user, password)
+        self.assertEqual(RealmAuditLog.objects.filter(event_time__gte=now).count(), 2)
```

Comment:
I would just test for the event_type here explictly, e.g. something like 
`self.assertEqual(RealmAuditLog.objects.filter(event_type='blah', event_time__gte=now).count(), 1)`